### ISSUE TITLE
Automatically generated doc is not correct

### ISSUE THREAD

**Issue by garro95** [rustsim/nalgebra#477]
In some points, there is some documentation generated automatically by some macro, that is not perfectly correct. A human should be able to see the meaning, but it can generate confusion at a first reading.
See for example https://nalgebra.org/rustdoc/nalgebra/base/struct.Matrix.html#method.sub_to. The doc says that the operation is equal to `self + rhs` while in fact it should perform `self - rhs`.

**Comment by sebcrozet** [comment]
That's right, the generated documentation is misleading. The macro generating this code [here](https://github.com/rustsim/nalgebra/blob/master/src/base/ops.rs#L370-L375) should take an additional parameter for the documentation.
